addons/mail/static/src/discuss/call/common/call_action_list.xml

```diff
@@ -6,7 +6,7 @@
             <div class="o-discuss-CallActionList-bar d-flex align-items-center flex-wrap justify-content-between" t-att-class="{ 'w-100 ps-2 pe-2': isSmall }">
                 <t t-if="isOfActiveCall and rtc.selfSession">
                     <t t-foreach="callActions.actions.slice(0, isMobileOS ? 3 : 4)" t-as="action" t-key="action_index">
-                        <t t-call="discuss.CallActionList.actionButton" />
+                        <CallActionButton action="action" isSmall="isSmall" isActive="action.isActive"/>
```

Comment:
`isActive` prop look useless: it's deduced from `action` prop.

Reply:
isActive doesn't work properly inside the child component.

Reply:
This shouldn't be the case. If this really happens then there's something wrong with not properly using OWL.reactive on `action` and this must be fixed or we will have some other problems.